cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/arcgis/NGSIArcgisFeatureTable.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2014-2017 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-cygnus (FIWARE project).
+ *
+ * fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-cygnus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+ */
+
+package com.telefonica.iot.cygnus.backends.arcgis;
+
+import java.util.Date;
+
+import com.telefonica.iot.cygnus.backends.arcgis.model.Feature;
+import com.telefonica.iot.cygnus.backends.arcgis.restutils.ArcgisFeatureTable;
+import com.telefonica.iot.cygnus.log.CygnusLogger;
+
+/**
+ * 
+ * @author PMO Santander Smart City – Ayuntamiento de Santander
+ *
+ */
+public class NGSIArcgisFeatureTable extends ArcgisFeatureTable {
+    private static final CygnusLogger LOGGER = new CygnusLogger(NGSIArcgisFeatureTable.class);
+    
+    private static final String CLASS_NAME = "NGSIArcgisFeatureTable";
+    private long timeoutSecs = 60;
+    private Date lastPersist = new Date();
+
+    /**
+     * 
+     * @param featureServiceUrl
+     * @param username
+     * @param password
+     * @param getTokenUrl
+     * @param b
```

Comment:
```suggestion
     * @param timeoutSecs
```

Weird... maybe there are another cases of "not fully replaced JavaDoc templates" like this?

Reply:
Fixed in 6cc4a5a